grails-app/migrations/0.8.x/changelog-2021-04-15-1340-insert-payment-term.xml

```diff
@@ -0,0 +1,95 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog/1.9"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog/1.9 http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-1.9.xsd">
+    <changeSet author="dhatala" id="1504202113400-2">
+        <preConditions onFail="MARK_RAN">
+            <sqlCheck expectedResult="0">
+                SELECT COUNT(*) FROM payment_term WHERE code = 'PP100';
+            </sqlCheck>
+        </preConditions>
+        <insert tableName="payment_term">
+            <column name="id" value="1"/>
+            <column name="version" valueNumeric="0"/>
+            <column name="code" value="PP100"/>
+            <column name="name" value="Prepayment 100"/>
+            <column name="description"/>
+            <column name="prepayment_percent" value="100"/>
+            <column name="days_to_payment"/>
+            <column name="date_created" valueDate="2021-04-15T00:00:00.0"/>
```

Comment:
I think you could declare:
`<property name="now" value="now()" dbms="mysql, postgresql"/>`
and then use `valueDate="${now}"` in each timestamp field's value, unless you really want to have this particular date here

Reply:
Ohh I like that suggestion.

Reply:
Where is that "now" property defined? Let's just support mysql for ... now. I don't want people looking at this code and thinking we support postgres until we actually fully support postgres.

Reply:
Properties are defined before changesets.

Reply:
@jmiranda seems like 1.9 version of liquibase does not support specifying dbms on properties yet. We'd need to upgrade liquibase to get this working in the future.